nasa-gibs/worldview
# WV-3242 index out of bounds error

## Description

> When adding more than one TEMPO layer, sometimes there is an "Index out of bounds" error displayed in the console

## How To Test

1. `git checkout WV-3242-index-out-of-bounds`
2. `npm ci && npm run build && npm run watch`
3. Go to this [link](http://localhost:3000/?v=-177.36395603910125,-25.17776302922428,-4.107158456019675,74.38473697077572&ici=3&icd=1&l=Coastlines_15m&lg=true&t=2024-07-02-T00%3A57%3A00Z)
4. Add several TEMPO L2 layers and verify that there are no `index out of bounds` errors.


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
